sql/pgwire_test.go

```diff
@@ -316,6 +313,103 @@ func TestPGPrepared(t *testing.T) {
 	}
 }
 
+func TestPGPreparedExec(t *testing.T) {
+	defer leaktest.AfterTest(t)
+	var base preparedTest
+	queryTests := []struct {
+		query string
+		tests []preparedTest
+	}{
+		{
+			"CREATE DATABASE d",
+			nil,
+		},
+		{
+			"CREATE TABLE d.t (i INT, s STRING, d INT)",
+			nil,
+		},
```

Comment:
Sort of strange that these are test cases. They are really part of test setup. I think it would be cleaner to pull them out and actually perform them as separate setup steps. You can do something like:

```
  setup := `CREATE DATABASE d; CREATE TABLE d.t (i INT, s STRING, d INT);`
  if err := db.Exec(setup); err != nil {
    t.Fatal(err)
  }
```